server/static/js/components.js

```diff
@@ -0,0 +1,136 @@
+Vue.component("robot-cardview", {
```

Comment:
This is the code managing the right side panel showing robot information and control buttons.